tox.ini

```diff
@@ -14,11 +14,13 @@ commands =
     coverage run -m unittest discover -v ./tests
     coverage run -a -m unittest discover -v ./ext/dapr-ext-grpc/tests
     coverage run -a -m unittest discover -v ./ext/dapr-ext-fastapi/tests
+    coverage run -a -m unittest discover -v ./ext/flask_dapr/tests
```

Comment:
shouldn't we follow the dapr-ext-* pattern for this directory too?

Reply:
unfortunately the package has always been called `flask_dapr` whereas the others are `dapr-ext-grpc` and `dapr-ext-fastapi`